Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN mkdir /etc/service/cromwell && \
     apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D && \
     echo 'deb https://apt.dockerproject.org/repo ubuntu-trusty main' > /etc/apt/sources.list.d/docker.list && \
     apt-get update && \
-    apt-get install yq docker-engine=1.8.2-0~trusty && \
+    apt-get -yq install docker-engine=1.8.2-0~trusty && \
```

Comment:
Sorry, I wasn't exactly clear.  😦 

What I tried to say too tersely was that there seem to be three different ways to attempt to express the standard automation options:

Our baseline Dockerfile:

`apt-get install yq ...`

This PR:

`apt-get -yq install ...`

The links above:

`apt-get install -yq ...`

The baseline variant seems incorrect, but it wasn't sure if the more common `apt-get install -yq` would be preferable to the changes here.